core/modules/wbase/Task.h

```diff
@@ -151,6 +151,8 @@ class Task : public util::CommandThreadPool {
     void setMemHandle(memman::MemMan::Handle handle) { _memHandle = handle; }
     void setMemMan(memman::MemMan::Ptr const& memMan) { _memMan = memMan; }
     void waitForMemMan();
+    bool getSafeToMoveRunning() { return _safeToMoveRunning; }
+    void setSafeToMoveRunning(bool val) { _safeToMoveRunning = val; }
```

Comment:
Is `setSafeToMoveRunning()` only meant for testing? Can you add a comment which says so?